ui/component/claimMenuList/view.jsx

```diff
@@ -158,7 +158,7 @@ function ClaimMenuList(props: Props) {
     if (!isRepost && !isChannel) {
       openModal(MODALS.CONFIRM_FILE_REMOVE, { uri });
     } else {
-      openModal(MODALS.CONFIRM_CLAIM_REVOKE, { claim, cb: !isRepost && (() => replace(`/$/${PAGES.CHANNELS}`)) });
```

Comment:
do you know why this was removed? I think it was to handle the re-direct after reposting.